app/_how-tos/operator-get-started-kic-install.md

```diff
@@ -29,8 +29,8 @@ prereqs:
   skip_product: true
 
 tldr:
-  q: How do I install {{ site.gateway_operator_product_name }} with {{ site.kic_product_name }} using Helm?
-  a: Update the Helm repository and use Helm to install {{ site.gateway_operator_product_name }} with {{ site.kic_product_name }}.
+  q: How do I install {{ site.operator_product_name }} with {{ site.kic_product_name }} using Helm?
+  a: Update the Helm repository and use Helm to install {{ site.operator_product_name }} with {{ site.kic_product_name }}.
```

Comment:
This would need to be rewritten as we don't deploy / install operator with KIC anymore.